can/interfaces/udp_multicast/bus.py

```diff
@@ -103,7 +102,9 @@ def __init__(
                 "receiving own messages is not yet implemented"
             )
 
-        super().__init__(channel, **kwargs)
+        super().__init__(
+            channel, **kwargs, protocol=CANProtocol.CAN_FD if fd else CANProtocol.CAN_20
+        )
 
         self.is_fd = fd
```

Comment:
@zariiii9003 Would you consider this `is_fd` flag as a public attribute that would lead to a breaking API change if it were removed? With the `protocol` field it's technically redundant and could be removed.

Reply:
I guess we could replace it with a property and raise a DeprecationWarning with a removal scheduled for version 5.